imaginary-iguanas/code_jam/users/validators.py

```diff
@@ -0,0 +1,19 @@
+from django.core.exceptions import ValidationError
+from datetime import date
+
+
+def calculate_age(born: date):
+    today = date.today()
+    return today.year - born.year - ((today.month, today.day) < (born.month, born.day))
```

Comment:
This is a nice utility function that should have been used in the `Profile` model as well.